src/main.cpp

```diff
@@ -33,6 +33,7 @@ int main(int argc, char** argv) {
     args::ValueFlag<std::string> appDirPath(parser, "appdir", "Path to target AppDir", {"appdir"});
 
     args::ValueFlagList<std::string> sharedLibraryPaths(parser, "library", "Shared library to deploy", {'l', "library"});
+    args::ValueFlagList<std::string> excludeLibraryPaths(parser, "library", "Shared library to exclude from deployment", {"exclude-library"});
```

Comment:
They're patterns, not paths, not filenames. This should be reflected in the help text.

```suggestion
    args::ValueFlagList<std::string> excludeLibraryPatterns(parser, "pattern", "fnmatch pattern used to exclude shared libraries from deployment", {"custom-exclude-library-pattern"});
```